rules-reviewed/eap7/eap6/eap6-xml.windup.xml

```diff
@@ -130,6 +130,24 @@
                 </iteration>
             </perform>
         </rule>
+        <rule id="eap6-xml-07000">
+            <when>
+               <xmlfile matches="/*[local-name()='ejb-jar']/*[local-name()='assembly-descriptor']/*[local-name()='clustering']/*[local-name()='clustered']" in="jboss-ejb3.xml"/>
+            </when>
+            <perform>
+                <iteration>
+                    <hint title="Stateful Session EJB Clustering changes in EAP 7" effort="1" severity="optional">
+                    <message>
+                        The clustering element is ignored in EAP 7 and is not useful. In case the application is started using HA profile, the replication will be done automatically.
```

Comment:
The check is good now but this text still refers to the entire `<clustering>` element.